english/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+from english.models import Class
+
+
+admin.site.register(Class)
```

Comment:
Is it work? Is it look nice? Do you have different filters and search here? Does the table with all classes look nice?